x-pack/plugin/core/src/main/resources/ilm-history.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
would it be clearer what this is if we renamed it to `ilm-history-index-template.json`? or something along the lines of including `-template` in the name?

Reply:
I followed the precedence from the SLM history implementation (`slm-history.json`), I think it's probably okay to leave this as-is though